14/umbraco-cms/tutorials/custom-error-page.md

```diff
@@ -36,19 +36,19 @@ This method will use a 404 page created via the backoffice.
 
 1. Navigate to the **Settings** section.
 2. Create a new "_Document Type with Template_".
-3. Name the Document Type **404 Page**.
-   1. Note: By default, Umbraco will generate the alias for this DocType as simply "Page," omitting the "404." You may want to manually adjust the alias to "404Page."
-4. \[Optional] Add properties on the Document Type.
+3. Name the Document Type **404**.
+4. Before saving the Document Type, adjust the alias to "Page404." Umbraco will not allow you to create a Document Type with a numeric alias.
```

Comment:
Just a tiny rewrite, to start the step with the _action_ word, "adjust" 😄 

```suggestion
4. Adjust the alias to "Page404" before saving the Document Type. Umbraco will not allow you to create a Document Type with a numeric alias.
```

Reply:
@sofietoft - I love it!